_posts/2023-05-10-solidity-0.8.20-release-announcement.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.20 Release Announcement'
+date: '2023-05-10'
+author: Solidity Team
+category: Releases
+---
+
+
+We are excited to announce the latest release of the Solidity Compiler, [Solidity v0.8.20](https://github.com/ethereum/solidity/releases/tag/v0.8.20). 
+This latest version includes a range of improvements in the via-IR pipeline and improves the list of events exposed in the contract ABI, and, most importantly, support for Shanghai!
```

Comment:
```suggestion
This latest version includes a range of improvements in the via-IR pipeline and improves the list of events exposed in the contract ABI, and, most importantly, support for the [Shanghai hard fork](https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/shanghai.md) !
```

For people who might now know immediately what Shanghai is.

Reply:
@ekpyron are you dismissing the "hard fork" bit, the link or both?

"support for Shanghai" sounds incomplete.